app/scripts/services/authentication.js

```diff
@@ -4,43 +4,42 @@ app.service('AuthenticationService', function ($http, $q, SessionStoreService) {
   return {
     login: function(user) {
       var deferred = $q.defer();
-      $http.post('//tma-develop.herokuapp.com/v1/sessions', user)
+      $http.post('//localhost:3000/v1/sessions', user)
```

Comment:
cool. So you are thinking of using an angular constant service for this? Or is there a better way?